receiver/netflowreceiver/README.md

```diff
@@ -0,0 +1,96 @@
+| ------------- |-----------|
+| Stability     | [development]: logs   |
+| Distributions | [contrib] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fnetflow%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fnetflow) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fnetflow%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fnetflow) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@evan-bradley](https://www.github.com/evan-bradley), [@dlopes7](https://www.github.com/dlopes7) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+The netflow receiver can listen for [netflow](https://en.wikipedia.org/wiki/NetFlow), [sflow](https://en.wikipedia.org/wiki/SFlow), and [ipfix](https://en.wikipedia.org/wiki/IP_Flow_Information_Export) data and convert it to OpenTelemetry logs. The receiver is based on the [goflow2](https://github.com/netsampler/goflow2) project.
+
+This gives Opentelemetry users the capability of monitoring network traffic, and answer questions like:
+
+* Which protocols are passing through the network?
+* Which servers and clients are producing the highest amount of traffic?
+* What ports are involved in these network calls?
+* How many bytes and packets are being sent and received?
+
+
+## Getting started
+
+By default the receiver will listen for ipfix and netflow on port `2055` and sflow on port `6343`. The receiver can be configured to listen on different ports and protocols.
+
+Example configuration:
+
+```yaml
+receivers:
+  netflow:
+    listeners:
+      - scheme: netflow
+        port: 2055
+        sockets: 16
+        workers: 32
+
+processors:
+  batch:
+    send_batch_size: 2000
+    timeout: 30s
+
+exporters:
+  debug:
+    verbosity: detailed
+
+service:
+  pipelines:
+    logs:
+      receivers: [netflow]
+      processors: [batch]
+      exporters: [debug]
+  telemetry:
+    logs:
+      level: debug
+```
+
+We recommend using the batch processor to reduce the number of log requests being sent to the exporter. The batch processor will batch log records together and send them in a single request to the exporter.
+
+You would then configure your network devices to send netflow, sflow, or ipfix data to the collector on the specified ports.
+
+## Configuration
```

Comment:
It would be good to include whether each option is required or optional, as well as the default value for each option (where relevant).

I see you mention some defaults under `Getting Started`, but it's good to include that information with the config options here.